src/ai/backend/manager/api/vfolder.py

```diff
@@ -2422,6 +2429,28 @@ async def clone(request: web.Request, params: Any, row: VFolderRow) -> web.Respo
     if source_proxy_name != target_proxy_name:
         raise InvalidAPIParameters("proxy name of source and target vfolders must be equal.")
 
+    async with root_ctx.db.begin_session() as sess:
+        if row["group"]:
+            log.debug("row[group]:{}", row["group"])
```

Comment:
Let's improve this logging.
Such as `VFolder group: {}`, or we can log it better!

Reply:
Thanks!
I will fix it.